packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -69,15 +75,19 @@ export default function DocumentActions( { template } ) {
 				'has-secondary-label': !! label,
 			} ) }
 		>
-			{ documentTitle ? (
+			{ entityTitle ? (
```

Comment:
This is a hella long ternary 😄  This is nitpicking but I think it would be more readable this way.
```js
{ ! entityTitle && __( 'Loading…' ) }
{ entityTitle && ... }
```

Reply:
+1
I didn't even notice it was a ternary while looking at the code! 😄

Reply:
true, I'll take care of that!

Reply:
fixed in 2ef39e0